src/hooks.rs

```diff
@@ -50,3 +51,20 @@ pub(crate) fn run_hook(
 
     Ok(())
 }
+
```

Comment:
Here you can add another function with `#[cfg(windows)]`  with same name.
So for example rename it to `run_script_file`

If on windows, the windows version will be included and the unix version will not. If on unix, only the unix version will be included. (and of course the windows version should just use the `Command::new` code)

Reply:
I didn't know if was allowed to have same fun name with conditional compilation.
I'll try.